src/fidesops/tasks/__init__.py

```diff
@@ -48,7 +48,7 @@ def _create_celery(config_path: str = config.execution.celery_config_path) -> Ce
 
 def start_worker() -> None:
     logger.info("Running Celery worker...")
-    celery_app.worker_main(argv=["worker", "--loglevel=info"])
+    celery_app.worker_main(argv=["worker", "--loglevel=info", "--concurrency=2"])
```

Comment:
This could be configurable in the future, but this matches our state before the move to Celery.  The default is the number of cores on your machine, but limiting the number of PrivacyRequests that can be run simultaneously per worker has a lot of positive effects including reducing the number of simultaneous connections on the application database, as well as the customers' owned databases, and reduces simultaneous requests against their connected API's.

Reply:
Thanks, I've added [a ticket](https://github.com/ethyca/fidesops/issues/957) to be addressed as a follow-up.